diff -Naur plank-0.6.0.1023.orig/data/plank.desktop.in plank-0.6.0.1023/data/plank.desktop.in
--- plank-0.6.0.1023.orig/data/plank.desktop.in	2014-05-04 22:53:34.000000000 +0700
+++ plank-0.6.0.1023/data/plank.desktop.in	2014-06-16 20:00:32.857545540 +0700
@@ -2,7 +2,7 @@
 Name=Plank
 GenericName=Dock
 _Comment=Plank. Stupidly simple.
-Categories=Utility;
+#Categories=Utility;
 Type=Application
 Exec=plank
 Icon=plank
